src/Oro/Bundle/OrganizationBundle/Form/Transformer/BusinessUnitTreeTransformer.php

```diff
@@ -26,7 +26,7 @@ public function __construct($manager)
     public function reverseTransform($value)
     {
         if (null === $value) {
-            return 0;
```

Comment:
I think this is typo error. Transformer should return value for model layer(BU entity or null), so integer type not allowed here.
Maybe here should be as 
```php
    public function reverseTransform($value)
    {
        if (null === $value) {
            $value = 0;
        } elseif (is_array($value)) {
```